### PR TITLE
Add question about why an extension is not installable

### PR DESCRIPTION
**Pull request by egamma**
This is the proposed text shown from the 'More Information' dialog when a user attempts to install a strict UI extension as requested in https://github.com/microsoft/vscode/issues/118104.

image

//fyi @sandy081

### CODE REVIEW

**Review by sandy081 — APPROVED** [review]
Thanks for the changes. LGTM

File: docs/remote/codespaces.md [hunk 1]
```diff
@@ -52,6 +52,12 @@ The VS Code extension API hides most of the implementation details of running re
 
 ## Common questions
 
+### Why is an extension not installable in the browser
+
+There is a small number of extensions that have built-in assumptions or needs to run on the desktop. For example, when an extension accesses files from the VS Code installation on the desktop or when an extension depends on a executable that must run in a desktop environement. Such extensions are tagged as 'UI' extensions by the extension author and when you install such an extension in the browser then you are informed that the extension is not available.
```
Comment (chrisdias) [inline docs/remote/codespaces.md:57]: There **are** a small number of extension...

**Review by chrisdias — CHANGES_REQUESTED** [review]
minor nit, one question in line.

**Comment by egamma** [comment]
reviewed with Chris.

**Pull request closed without merging**